Makefile

```diff
@@ -129,10 +129,20 @@ define yq_get_db
     $(shell $(call yq_get, .db.$(1)))
 endef
 
+DB_USER ?= $(strip $(call yq_get_db,user))
+DB_PASSWORD ?= $(strip $(call yq_get_db,password))
+DB_HOST ?= $(strip $(call yq_get_db,host))
+DB_PORT ?= $(strip $(call yq_get_db,port))
+DB_NAME ?= $(strip $(call yq_get_db,dbname))
+
 define database_url
-	postgres://$(strip $(call yq_get_db,user)):$(strip $(call yq_get_db,password))@$(strip $(call yq_get_db,host)):$(strip $(call yq_get_db,port))/$(strip $(call yq_get_db,dbname))?sslmode=disable
+	postgres://$(DB_USER):$(DB_PASSWORD)@$(DB_HOST):$(DB_PORT)/$(DB_NAME)?sslmode=disable
 endef
 
+.PHONY: create-postgres-db
+create-postgres-db: ## create database specified in the config file.
+	PGPASSWORD=$(DB_PASSWORD) psql -h $(DB_HOST) -p $(DB_PORT) -U $(DB_USER) -c "CREATE DATABASE $(DB_NAME);"
+
```

Comment:
Will this work if `psql` is not installed in the environment? I think we should add the `psql` installation under the `install-dependencies` command.

Reply:
Makes sense, I moved away from psql to avoid installing it and used `values.yaml` file when setting up postgres to create db right away